hd/lang/lex_utf8.txt

```diff
@@ -28,6 +28,10 @@ fr: dictionnaire des %s
 en: compiled on
 fr: compilé le
 
+    commited on
```

Comment:
It's two différents translations compiled/commited. I suggested this night to link directly the version to the commit on Github where date is also/already given: `<a href="https://github.com/geneweb/geneweb/commit/$commit;" title="Commit $commit; $commit_date;">$commit;</a>` but I did not manage the %compilation_time; in this and left it in its bvar after $version;